src/sqlancer/tidb/gen/TiDBAnalyzeTableGenerator.java

```diff
@@ -31,12 +31,13 @@ public static SQLQueryAdapter getQuery(TiDBGlobalState globalState) throws SQLEx
             sb.append(Randomly.getNotCachedInteger(1, 1024));
             sb.append(" BUCKETS");
         }
-        return new SQLQueryAdapter(sb.toString(), ExpectedErrors.from("https://github.com/pingcap/tidb/issues/15993",
-                /* https://github.com/pingcap/tidb/issues/15993 */ "doesn't have a default value" /*
-                                                                                                   * https://github.
-                                                                                                   * com/pingcap/tidb/
-                                                                                                   * issues/15993
-                                                                                                   */));
+        return new SQLQueryAdapter(sb.toString(),
+                ExpectedErrors.from("https://github.com/pingcap/tidb/issues/15993",
+                        /* https://github.com/pingcap/tidb/issues/15993 */ "doesn't have a default value",
+                        "Index 'PRIMARY' in field list does not exist in table" /*
+                                                                                 * https://github. com/pingcap/tidb/
+                                                                                 * issues/15993
+                                                                                 */));
```

Comment:
```suggestion
        return new SQLQueryAdapter(sb.toString(),
                ExpectedErrors.from("https://github.com/pingcap/tidb/issues/15993", "doesn't have a default value", "Index 'PRIMARY' in field list does not exist in table"));
```

The suggestion seems to not select the right code block, hope you can get it :)